README.md

```diff
@@ -19,12 +19,24 @@ config = {
     'LINK' => DraftjsExporter::Entities::Link.new(className: 'link')
   },
   block_map: {
-    'header-one' => { element: 'h1' },
+    'header-one' => { element: 'h1', attrs: { id: 'foo-bar' }},
```

Comment:
Not necessary, but it would have been nice to have this syntax match the one used by `wrapper`:

```ruby
wrapper: ['ul', { className: 'public-DraftStyleDefault-ul' }]
```